### PR TITLE
[Utopia-764] subheader for pia

### PR DESCRIPTION
**Pull request by binggaofw**
# Description

This PR includes the following proposed change(s):

- Add subheader component to pia 
there are many scenarios. so I just post the screenshot here for refer

Screen Shot 2023-02-10 at 5 02 27 PM

Screen Shot 2023-02-10 at 5 01 47 PM

Screen Shot 2023-02-10 at 4 59 19 PM

Screen Shot 2023-02-10 at 4 58 43 PM

Screen Shot 2023-02-10 at 4 56 09 PM

add subheader to detail page
Screen Shot 2023-02-13 at 9 20 43 AM

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

**Comment by binggaofw** [comment]
@kushal-arora-fw  @Bottle7  @antsand  please help to test in your local for functionalities.

File: src/frontend/src/components/public/PIASubHeader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      setIsDownloading(false);
+    }
+  };
+  const handleAlertClose = () => {
+    setDownloadError('');
+  };
+  return (
+    <>
+      <div className="subheader-container wrapper">
+        <h1>{pia.title ? pia.title : 'New PIA'}</h1>
```
Comment (antsand) [inline src/frontend/src/components/public/PIASubHeader/index.tsx:54]: Should this be untitled?

File: src/frontend/src/components/public/PIASubHeader/_piasubheader.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.subheader-container {
+    display: flex;
+    position: relative;
+    top: 150px;
```
Comment (antsand) [inline src/frontend/src/components/public/PIASubHeader/_piasubheader.scss:4]: May want to assign this to a sass variable

**Review by antsand — DISMISSED** [review]

File: src/frontend/src/components/public/PIASubHeader/index.tsx (hunk 1 above)
Comment (kushal-arora-fw) [reply src/frontend/src/components/public/PIASubHeader/index.tsx:54]: is this default text "New PIA" explicitly mentioned in the ticket?

File: src/frontend/src/pages/PIAIntakeForm/messages.ts [hunk 3]
```diff
@@ -124,10 +124,10 @@ export default {
         en: `Cancel`,
       },
       TitleText: {
-        en: `Submit for MPO review?`,
+        en: `Submit for PIA Intake?`,
       },
       ParagraphText: {
-        en: `Your PIA status will be changed to MPO review. Your MPO will be notified that your edits are complete and your PIA is ready for review.`,
+        en: `Are you sure you want to submit your PIA intake?`,
```
Comment (kushal-arora-fw) [inline src/frontend/src/pages/PIAIntakeForm/messages.ts:130]: Same here. Is this change in text mentioned in the ticket?

Comment (binggaofw) [reply src/frontend/src/pages/PIAIntakeForm/messages.ts:130]: Not in the ticket itself, but in the figma design mock up.

**Review by kushal-arora-fw — COMMENTED** [review]
image

- Looking a little weird on my mac.
- Breaks on scrolling; I remember this needs to be fixed below header

**Comment by binggaofw** [comment]
@kushal-arora-fw I fix the css issue, please take a look, it fixed position now and not flow to nav bar

**Review by kushal-arora-fw — COMMENTED** [review]
Still doesn't match designs. Spacing issue. 

image

image


EXPECTED:
image

**Review by kushal-arora-fw — APPROVED** [review]

**Pull request merged**